tensornetwork/backends/pytorch/pytorch_backend.py

```diff
@@ -69,9 +69,12 @@ def rq_decomposition(
   ) -> Tuple[Tensor, Tensor]:
     return decompositions.rq_decomposition(self.torch, tensor, split_axis)
 
-  def concat(self, values: Tensor, axis: int) -> Tensor:
+  def shape_concat(self, values: Tensor, axis: int) -> Tensor:
     return np.concatenate(values, axis)
 
+  def concat(self, values: Tensor, axis: int = 0) -> Tensor:
+    return np.stack(values, axis)
```

Comment:
Aw, that would explain it. You'll need to use pytorch's concat equivalent here instead of numpy.